oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -12,11 +16,100 @@ def __init__(self, project_item):
+
+    def _populate_language(self):
+        for ilu in ItemLanguageUsage.objects.filter(item=self._item):
+            lang = mods.Language()
+            lang.terms.append(mods.LanguageTerm(text=ilu.value))
+            self.languages.append(lang)
+
+    def _populate_format(self):
+        format = Format.objects.filter(item=self._item).first()
+        if format:
+            self.create_physical_description()
+            self.physical_description.extent = format.value
+
+    def _populate_description(self):
+        descriptions = Description.objects.filter(item=self._item)
+
+        # Similar note element behaving qualifiers
+        type_labels = {
+            "biographicalnote": "Biographical Information",
+            "interviewerhistory": "Interviewer Background and Preparation",
+            "personpresent": "Persons Present",
+            "place": "Place Conducted",
+            "processinterview": "Processing of Interview",
+            "supportingdocuments": "Supporting Documents",
+        }
+
+        for desc in descriptions:
+            desc_type = desc.type.type.lower()
+
+            if desc_type == "abstract":
+                self.create_abstract()
+                self.abstract.text = desc.value
+
+            elif desc_type == "adminnote":
+                self.notes.append(self._parse_adminnote(desc.value))
+
+            elif desc_type in type_labels.keys():
+                type_value = (
+                    "biographical" if desc_type == "biographicalnote" else desc_type
+                )
+                self.notes.append(
+                    mods.Note(
+                        type=type_value, label=type_labels[desc_type], text=desc.value
+                    )
+                )
+
+            else:
+                self.notes.append(mods.Note(text=desc.value))
+
+    def _parse_adminnote(self, desc_value: str) -> mods.Note:
+        note_components = desc_value.split(":", 1)
+
+        if len(note_components) <= 1:
+            return mods.Note(text=desc_value)
+
+        else:
+            admin_note_type = note_components[0].lower()
+            admin_note_value = note_components[1]
+
+            if admin_note_type == "supporting documents":
+                return mods.Note(
+                    label="Supporting Documents",
+                    type="supportingdocuments",
+                    text=admin_note_value,
+                )
+
+            elif admin_note_type == "interviewer background and preperation":
+                return mods.Note(
+                    label="Interviewer Background and Preperation",
+                    type="interviewerhistory",
+                    text=admin_note_value,
+                )
+
+            elif admin_note_type == "processing of interview":
+                return mods.Note(
+                    label="Processing of Interview",
+                    type="processinterview",
+                    text=admin_note_value,
+                )
```

Comment:
There's no `else`, so only admin notes with 1 of these 3 types (or no type) will be added.  I checked our data, and of our 582 description records with type=adminnote:
* 288 have no colon, so would return the full value; that's fine
* 294 have a colon, so would get `admin_note_type` checked
* only 1 of those 294 would return a value, for "supporting documents"

Example: http://127.0.0.1:8000/item/9603 (21198/zz002kc9zw).

See https://gist.github.com/akohler/505cb62e2dc51e0320f3983d85ca9201 for the code and output I get.

Reply:
You are correct, there should be an `else` upon review of the [original logic](https://github.com/UCLALibrary/DLDataProvider/blob/81a60ab4fc668590d276a190e1068ed1f4f79cc6/Model/src/edu/ucla/library/digital/jaxb/generateModsXML.java#LL250C21-L250C21)

Reply:
Per discussion with @emcaulay - `Description`s of `DescriptionType` with value `adminnote` will not be displayed in the exported MODS record